packages/shared/src/components/cards/WelcomePostCardFooter.tsx

```diff
@@ -21,12 +24,20 @@ export const WelcomePostCardFooter = ({
       />
     );
   }
-
   if (post.content) {
     return (
-      <p className="px-2 break-words line-clamp-6 typo-callout">
-        {post.content}
-      </p>
+      <>
+        <p className="px-2 break-words line-clamp-6 typo-callout">
+          {contentRef?.current?.textContent}
```

Comment:
The main point is that this is not reactive so it feels clunky. Could you not just use result of `sanitize` directly?

Reply:
Hmm but would you not be able to use `FORBID_TAGS` from the lib to remove `img`? You are currently using `textContent` for that right?

Reply:
Not just the images, but all the noise in the content for example when making a heading which requires a `#` character or a bold text from using `*`.

Reply:
Haven't heard of `FORBID_TAGS` though. Care to elaborate?

Reply:
It is part of this DOMPurify lib we are using for `sanitize`, check in the examples here https://github.com/cure53/DOMPurify#can-i-configure-dompurify

Reply:
Basically you should be able to specific tags that will be removed from HTML string which is exactly what we do here.

Reply:
Oh, nice. That is probably a better option to set `ALLOWED_TAGS` to an empty array so there will be no rendered tag. Will try this once I get back on it a bit later.

Reply:
We basically want no tag since it will affect the card's layout. So `ALLOWED_TAGS` with empty content would probably our best bet.